subgraph.template.yaml

```diff
@@ -12,7 +12,7 @@ dataSources:
       startBlock: {{creationBlockNumber}}
     mapping:
       kind: ethereum/events
-      apiVersion: 0.0.4
+      apiVersion: 0.0.5
```

Comment:
any reason not to use 0.0.6? 
also, the specVersion is now 0.0.4 , maybe we should update

not mandatory at all